reloc.ml

```diff
@@ -251,7 +251,16 @@ let quote_files cmdline lst =
 let cygpath l cont =
   let accept_error = (!use_cygpath = `Try && l <> []) in
   let l =
-    let args = String.concat " " (List.map Filename.quote l) in
+    let s = String.concat " " (List.map Filename.quote l) in
+    let args =
+      if String.length s >= 1024 then begin
+        let (fn, oc) = open_temp_file "cygpathargs" "" in
```

Comment:
cf. https://github.com/cygwin/cygwin/blob/main/winsup/utils/cygpath.cc#L1081 - useful, because the `?perms` argument is 4.03+ 🫣
```suggestion
        (* cygpath loads the file in "text" mode, so CRLF endings are fine *)
        let (fn, oc) = open_temp_file "cygpathargs" "" in
```